// components/TestError.jsx

const TestError = () => {
  throw new Error('Deliberate error for testing!');
};

export default TestError;
